src/rpc/mining.cpp

```diff
@@ -1368,7 +1368,7 @@ UniValue finalizecompactblock(const JSONRPCRequest& request)
 
     // Cached transactions
     std::vector<unsigned char> found_tx(ParseHex(request.params[2].get_str()));
-    CDataStream ssFound(block_tx, SER_NETWORK, PROTOCOL_VERSION);
+    CDataStream ssFound(found_tx, SER_NETWORK, PROTOCOL_VERSION);
```

Comment:
Woaaa, that's a nasty one!

Reply:
yeah pretty annoying to chase down, print statements everywhere found it